packages/driver/src/cy/chai.js

```diff
@@ -97,6 +97,18 @@ chai.use((chai, u) => {
     if (val && (getType(val) === 'Window')) {
       return '[window]'
     }
```

Comment:
can you replace this logic with the try-catch as well? I think the try-catch is a better strategy

Reply:
I realized that this `if` statement can be removed entirely and try-catch can handle it, too. So, I removed it and `getType`.